.github/workflows/build.yml

```diff
@@ -56,11 +56,11 @@ jobs:
           opt_level: avx2
   linux-x86_64-AVX512-cmake:
     name: Linux x86_64 AVX512 (cmake)
-    if: false # TODO: enable when GitHub Actions adds AVX-512 hosts
-    needs: linux-x86_64-cmake
-    runs-on: ubuntu-latest
+    continue-on-error: true # non-blocking mode for now
+    runs-on: faiss-aws-m7i.xlarge
     steps:
       - name: Checkout
+        continue-on-error: true # non-blocking mode for now
```

Comment:
Do we need this line here?

Reply:
I observed a difference in UI behaviors for `continue-on-error` when set on a job vs on a step: setting it on a job means the action is successful but people will still see the red check mark on the github PR page. I wanted the red check mark to go away to avoid confusion and that means we also need `continue-on-error` set on the step level. There has been a discussion about this behavior in: https://github.com/orgs/community/discussions/15452

Reply:
Interesting. So it's different when it's set at job level and the step level. Do we need to set it on the build_cmake as well then?

Reply:
Ah nice catch!